### DISCUSSION TITLE
Piper on a raspberry pi Zero 2 W?

### DISCUSSION THREAD

**Discussion by Yobeco** [rhasspy/piper#621] [Q&A]
Hi! :-) 
Great application!
Is it possible to run Piper on a raspberry pi Zero 2 W running Raspbian Lite?
I need to build a lightweight device that consumes very low levels of power.

Thank you

**Comment by zoldaten** [comment]
possible

**Comment by lcraver** [comment]
Did yall ever test this?

**Reply by zoldaten** [reply]
i tested. it is far from realtime but works. and there's problem with sound output in headless mode but it could be solved.

**Reply by lcraver** [reply]
How far from realtime is it? I expected it to be pretty slow but still within a couple seconds hopefully for a sentence?
